src/Symfony/Component/Translation/Resources/bin/translation-status.php

```diff
@@ -162,7 +162,7 @@ function extractLocaleFromFilePath($filePath)
 function extractTranslationKeys($filePath): array
 {
     $translationKeys = [];
-    $contents = new \SimpleXMLElement(file_get_contents($filePath));
+    $contents = new SimpleXMLElement(file_get_contents($filePath));
```

Comment:
The `\` is removed because this file is lacking a namespace declaration and thus runs inside the global namespace, right? Curious that constants like `\PHP_SAPI` or `\PHP_EOL` are not changed.